src/docs/releases/3.0.0.md

```diff
@@ -6,6 +6,14 @@ Before upgrading from version 2 to v3, it is important to first compile your pro
 
 ## Breaking Changes
 
+### Email Module
+
+Previously, emails sent from Orchard Core could have either a plain text body, or an HTML body, but not both. Now, they can have both. This also brings some code-level API changes, see below.
+
+When interacting with email-related services from code, `MailMessage`, the class representing an e-mail, exposed a `string` `Body` property. This could contain either plain text or HTML, which was indicated by `IsHtmlBody`.
+
+While this API is still available, it's deprecated in favor of the new `Body` property that uses the new `MailMessageBody` type. This can contain a plain text and/or HTML body.
```

Comment:
This is not true, you've removed the old API. But it would be actually better to deprecate the old API first, as described here.